keyboards/minidox/keymaps/dustypomerleau/config.h

```diff
@@ -0,0 +1,30 @@
+// dustypomerleau, Minidox config
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant. 

```suggestion
```